.jshintrc

```diff
@@ -8,6 +8,6 @@
 	"nonbsp": true,
 	"undef": true,
 	"unused": true,
-	"strict": "global",
+	"strict": "implied",
 	"varstmt": true
 }
```

Comment:
There is also `.jshintrc` file in `dev/` directory that should be changed. We can use `implied` there too I think.